resources/cmake/FindRTIConnextDDS.cmake

```diff
@@ -148,6 +182,54 @@
 #   - ``MONITORING_LIBRARIES``
 #     (e.g., ``MONITORING_LIBRARIES_RELEASE_STATIC``)
 #
+# - ``discovery_static``:
+#   - ``DISCOVERY_STATIC``
+#     (e.g., ``DISCOVERY_LIBRARIES_RELEASE_STATIC``) TODO: add more info??
+#
+# - ``nddstls``:
+#   - ``NDDSTLS``
+#     (e.g., ``NDDSTLS_LIBRARIES_RELEASE_STATIC``) TODO: add more info??
+#
+# - ``metp``:
+#   - ``METP``
+#     (e.g., ``METP_LIBRARIES_RELEASE_STATIC``) TODO: add more info??
+#
+# - ``transport_tcp``:
+#   - ``TRANSPORT_TCP``
+#     (e.g., ``TRANSPORT_TCP_LIBRARIES_RELEASE_STATIC``) TODO: add more info??
```

Comment:
Add also the "<lib> libraries if found"

Reply:
Isn't that written in the imported targets documentation section? Do you want me to write it also here?